integration_test/third_party_apps_test/applications/postgresql/metadata.yaml

```diff
@@ -189,9 +189,9 @@ expected_logs:
       - name: jsonPayload.tid
         type: number
         description: Thread ID where the log originated
-      - name: jsonPayload.role
+      - name: jsonPayload.database
```

Comment:
[Optional] I wonder if it makes sense to move this after `jsonPayload.user`, to match the order of the fields in the regex?